### PR TITLE
Badge status

### PR DESCRIPTION
**Pull request by rukawa444**
This PR fixes the badge of projects with a cypress test.

badge
badged

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-12)

**Comment by rukawa444** [comment]
> Go go go 😄

Thank you! you're always in my prayers🙌😎 lol!

File: features/projects/types/project.types.ts [hunk 1]
```diff
@@ -5,9 +5,9 @@ export enum ProjectLanguage {
 }
 
 export enum ProjectStatus {
-  stable = "stable",
+  info = "info",
   warning = "warning",
-  critical = "critical",
+  error = "error",
```
Comment (jkettmann) [inline features/projects/types/project.types.ts:10]: Great job detecting the problem correctly! That's the main challenge in this task. I've seen complicated solutions here, so honestly 👍

File: cypress/integration/badge-status.spec.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+describe("Badge Status", () => {
```
Comment (jkettmann) [inline cypress/integration/badge-status.spec.ts:1]: Great job at implementing this test. I'd probably move this to the `project-list.spec` file. Seems to belong there from my perspective

File: features/projects/types/project.types.ts (hunk 1 above)
Comment (rukawa444) [reply features/projects/types/project.types.ts:10]: This is actually very difficult!! haha! but it in the end, I thought this is the better solution.

File: cypress/integration/badge-status.spec.ts (hunk 2 above)
Comment (cathleys) [reply cypress/integration/badge-status.spec.ts:1]: I have a question, Should I need to create a new PR for this? Thanks!

**Pull request merged**